topics/discord-bots/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: discordbots
+topic: discord-bots
+github_url: https://github.com/discord
+display_name: Discord Bots (Extensions)
+logo: discord-bots.png
+short_description: Discord Bots are extensions for the Discord chat app.
+url: https://top-bots.xyz
```

Comment:
I don't feel comfortable linking to a 3rd party bot listing site here. Can we remove this line?
```suggestion
```

Reply:
Let's wait for the discord to release its own listing service (it will do it in about a month or two)

Reply:
Or we can for now replace it with https://discord.dev, which is an official documentation of discord's api

Reply:
sure, yeah that works!
```suggestion
url: https://discord.dev
```